emloop/tests/hooks/classification_metrics_test.py

```diff
@@ -0,0 +1,51 @@
+"""
+Test module for computing epoch statistics for classification tasks hook (emloop.hooks.classification_metrics).
+"""
+import collections
+import os
+
+import numpy as np
+import pytest
+
+from emloop.hooks.classification_metrics import ClassificationMetrics
+
+
+def get_epoch_data():
+    epoch_data = collections.OrderedDict([
+        ('train', collections.OrderedDict([
+            ('gt', np.array([0, 0, 0, 0, 1])),
+            ('prediction', np.array([0, 0, 0, 1, 1])),
+            ('accuracy', 1),
+        ]))
+    ])
+    return epoch_data
+
+
+@pytest.mark.skipif(os.environ.get('EXTRA_PKGS', None) is None, reason='This test requires SciKit.')
+def test_computing_metrics():
+    """Test metrics is correctly computed."""
+
+    prefix = 'metrics_'
+    hook = ClassificationMetrics('prediction', 'gt', 'binary', prefix)
+    epoch_data = get_epoch_data()
+
+    hook.after_batch(stream_name='train', batch_data=epoch_data['train'])
```

Comment:
hm,, it would be better to use `after_batch` at least twice so that we know, that it really computes will data from all batches